src/maintagent/ui/maintagent.php

```diff
@@ -93,7 +93,8 @@ function DisplayForm()
                      "D"=>_("Vacuum Analyze the database."),
               //       "U"=>_("Process expired uploads (slow)."),
               //       "Z"=>_("Remove orphaned files from the repository (slow)."),
-                     "v"=>_("verbose (turns on debugging output)")
+                     "v"=>_("verbose (turns on debugging output)"),
+                     "I"=>_("Reindexing of database (This activity may take 5-10 mins. Execute only when system is not in use).")
```

Comment:
Just a minor change: please reorder such that `"v" =>...` is the last item. (after that I will merge this PR)